src/ocaml/merlin_specific/tast_helper.ml

```diff
@@ -14,15 +14,8 @@ module Pat = struct
       | None -> str.Asttypes.loc
       | Some loc -> loc
     in
-<<<<<<< HEAD
     let mode = Mode.Value.newvar () in
-    let uid = Uid.mk ~current_unit:(Env.get_unit_name ()) in
```

Comment:
Do you know the story behind this line? It looks like it was added in the 4.14.1-19 merge, but was never added upstream.

Reply:
Upstream added `uid` as a parameter to `var`. That's why I can delete this line now.